docs/howto/openstack/neutron/new-network-control-panel.md

```diff
@@ -0,0 +1,58 @@
+# Creating a new network from the Cleura Cloud Control Panel
+
+Before creating a server in {{extra.brand}} Cloud, you need to have a 
+network to make the new server a member of. You may have more than one 
+network per region, so let us now walk through creating a new network 
+using the {{extra.brand}} Cloud Control Panel. Fire up your favorite web 
+browser, navigate to the [Cleura Cloud](https://cleura.cloud) page, and 
+login into your {{extra.brand}} account.
+
+On the top right-hand side of the {{extra.brand}} Cloud Control Panel, 
+click the _Create_ button. A new pane will slide into from the 
+right-hand side of the browser window, titled ---unsurprisingly 
+enough--- _Create_.
+
+![Create a new object](assets/new-net-panel/shot-01.png)
+
+You will notice several rounded boxes prominently displayed on that 
+pane, each for defining, configuring, and instantiating a different 
+{{extra.brand}} Cloud object. Go ahead and click the _Network_ box. A 
+new pane titled _Create Network_ will slide over. At the top, type in a 
+name and select one of the available regions for the new network.
+
+![New network name and region](assets/new-net-panel/shot-02.png)
+
+Expand the _Advanced Options_ section below and make sure _Port 
+Security_ is enabled. You do not have to set a value for the MTU 
```

Comment:
I would suggest to say "Leave the MTU parameter blank". That way the phrasing is affirmative rather than negative, and the recommendation is stronger than saying "you don't have to" (which implies "... but you can", and in most circumstances people really should *not* override the default MTU size).

Reply:
Much better to follow your approach, yes.